.devcontainer/create_cert

```diff
@@ -2,7 +2,7 @@
 
 # create self-signed certificate for testing with the browser
 
-HOST=data.ghga-dev.de
+HOST=$(echo ${REACT_APP_CLIENT_URL} | awk -F[/:] '{print $4}')
```

Comment:
This will break if REACT_CLIENT_URL is not (yet) set in local.env.

I suggest to either revert it using a fixed default. Or to set a default in docker_compose.yml.

Currently we need to build twice because the local.env is created on startup. One idea would be to use initializeCommand to create the env file, before the container is started. But let's try not to fix all of this in this PR.